src/jit/assertionprop.cpp

```diff
@@ -808,10 +808,13 @@ Compiler::AssertionDsc* Compiler::optGetAssertion(AssertionIndex assertIndex)
  * if they don't care about it. Refer overloaded method optCreateAssertion.
  *
  */
-AssertionIndex Compiler::optCreateAssertion(GenTree* op1, GenTree* op2, optAssertionKind assertionKind)
+AssertionIndex Compiler::optCreateAssertion(GenTree*         op1,
```

Comment:
I added comment headers to all affected assertionprop functions. However, I also cleaned up things a bit because I don't want to waste my time adding comments about useless overloads and parameters - `assertion` parameter isn't needed because everyone just passes the address of a local variable created solely for this purpose.

Of course, it would be even better to split `optCreateAssertion` into smaller functions such that `helperCallArgs` parameter isn't needed but that doesn't belong in this PR.